zest/releaser/pypi.py

```diff
@@ -293,14 +310,22 @@ def extra_message(self):
             [zest.releaser]
             extra-message = [ci skip]
         """
-        default = ""
-        if self.config is None:
-            return default
-        try:
-            result = self._get_text("zest.releaser", "extra-message", default=default)
-        except (NoSectionError, NoOptionError, ValueError):
-            return default
-        return result
+        return self.__get_message_config__("extra-message")
+
+    def prefix_message(self):
+        """Return extra text to be added before the commit message.
+
+        This can for example be used to skip CI builds.  This at least
+        works for Travis.  See
+        http://docs.travis-ci.com/user/how-to-skip-a-build/
+
+        Enable this mode by adding a ``prefix-message`` option, either in the
+        package you want to release, or in your ~/.pypirc::
+
+            [zest.releaser]
+            prefix-message = [ci skip]
```

Comment:
This doesn't make sense. You're extracting the actual code into a separate function (instead of duplicating it) and instead you duplicate the docstring of the method. And in a way that doesn't apply as ci-skip is already handled by the other method.

Replace it with a proper rationale, please.

Reply:
@reinout Should be fixed by https://github.com/zestsoftware/zest.releaser/pull/389#discussion_r965366665